rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/360_date_histogram.yml

```diff
@@ -61,3 +112,103 @@ setup:
   - match: { aggregations.histo.buckets.8.doc_count: 1 }
   - match: { aggregations.histo.buckets.12.key_as_string: "2016-06-01T00:00:00.000Z" }
   - match: { aggregations.histo.buckets.12.doc_count: 1 }
+
+---
+"date_histogram and date_histogram_composite timezone":
+  - skip:
+      version: " - 8.1.99"
+      reason: bug fixed in 8.2.0
+  - do:
+      search:
+        index: test_timezone
+        body:
+          size: 0
+          aggs:
+            date_histogram_simple_yes_tz:
+              date_histogram:
+                field: date
+                format: "yyyy-MM-dd HH:mm:ss"
+                calendar_interval: hour
+                time_zone: Asia/Jakarta
+            date_histogram_simple_no__tz:
+              date_histogram:
+                field: date
+                format: "yyyy-MM-dd HH:mm:ss"
+                calendar_interval: hour
+            date_histogram_compos_yes_tz:
```

Comment:
The composite version should be in the composite test file, `230_composite.yml`, please.